service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -68,26 +73,10 @@ public OrderTypeAndDocument buildOrderTypeAndDocument(OrderTypeAndDocument typeA
             .build();
     }
 
-    /**
-     * Method to populate the order based on type of order selected
-     * Adds/formats the order title/details for C21 and the expiry date for supervision order
-     * Always adds order type, document, {@link JudgeAndLegalAdvisor} object and a formatted order date.
-     *
-     * @param typeAndDocument      the type of the order and the order document (document only shown in check answers)
-     * @param order                this value will contain fixed details and document values as well as customisable
-     *                             values.
-     * @param judgeAndLegalAdvisor the judge and legal advisor for the order.
-     * @param orderMonths          the number of months the supervision order is valid
-     * @param interimEndDate       the end date wrapper for an interim order
-     * @return Element containing randomUUID and a fully populated order, ready to be added to orderCollection.
-     */
-    public Element<GeneratedOrder> buildCompleteOrder(OrderTypeAndDocument typeAndDocument,
-                                                      GeneratedOrder order,
-                                                      JudgeAndLegalAdvisor judgeAndLegalAdvisor,
-                                                      LocalDate dateOfIssue,
-                                                      Integer orderMonths,
-                                                      InterimEndDate interimEndDate) {
-        GeneratedOrder generatedOrder = defaultIfNull(order, GeneratedOrder.builder().build());
+    public GeneratedOrder buildCompleteOrder(OrderTypeAndDocument typeAndDocument,
```

Comment:
ANy reason you decided to remove comment? No longer valid? Difficult to maintain? It's ok, better to remove than say wrong things!